### ISSUE TITLE
Copy button is not working in Chrome based browsers

### ISSUE THREAD

**Issue by agazso** [felfele/swapchat#1]
Tried it in
- Chrome
- Brave
- Edge

**Comment by tabcat** [comment]
is there a reason to not use https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText ?

**Comment by agazso** [comment]
According to the docs that that does not work in Edge and Safari :)

The best would be to use some clipboard polyfill. With a quick search I found this and it looks promsing, but I haven't tried and have not to much experience with browser polyfills.

https://github.com/lgarron/clipboard-polyfill

Maybe it's worth a try.

**Comment by tabcat** [comment]
I will try this out and can make a PR if i get it working

**Comment by tabcat** [comment]
live instance: https://swarm-gateways.net/bzz:/ea6a835c1f76cf95937336e5dfe8aa9e7557c8ba6e004d60e73ab002a668a0db/

I decided not to use the package because I think I found a much simpler solution which I've only been able to test in chromium, chrome and firefox.
https://github.com/tabcat/swapchat/blob/4f619ea156d04b2be17ff85b336ebfff36cf8370/dist/index.html#L20
If input.select exists and doesn't work in the other browsers then the polyfill will probably have to be used. There was no support documentation for browsers that I could find. https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/select
